table/server/master/src/test/java/alluxio/master/table/AlluxioCatalogTest.java

```diff
@@ -326,18 +326,18 @@ public void getTransformPlanOutputUri() throws Exception {
     ServerConfiguration.set(PropertyKey.MASTER_HOSTNAME, "localhost");
     ServerConfiguration.set(PropertyKey.MASTER_RPC_PORT, "8080");
     List<TransformPlan> plans = mCatalog.getTransformPlan(dbName, tableName, TRANSFORM_DEFINITION);
-    assertEquals("alluxio://localhost:8080/",
+    assertEquals("alluxio://",
```

Comment:
Isn't this supposed to have 3 slashes at the end, for an empty authority? In the previous value, there were 3 slashes...